src/utils.tsx

```diff
@@ -1,17 +1,26 @@
 import React, { Children, ReactNode } from 'react';
-import { StyleSheet, View } from 'react-native';
+import { View } from 'react-native';
 
-export const childrenWithOverriddenStyle = (children?: ReactNode) => {
+export const childrenWithOverriddenStyle = (
+  children?: ReactNode,
+  pageMargin = 0
+) => {
   return Children.map(children, (child) => {
     const element = child as React.ReactElement;
     return (
       // Add a wrapper to ensure layout is calculated correctly
-      <View style={StyleSheet.absoluteFill} collapsable={false}>
-        {/* @ts-ignore */}
+      <View
+        style={{
+          height: '100%',
+          width: '100%',
+          paddingHorizontal: pageMargin / 2,
+        }}
+        collapsable={false}
+      >
         {React.cloneElement(element, {
           ...element.props,
           // Override styles so that each page will fill the parent.
-          style: [element.props.style, StyleSheet.absoluteFill],
+          style: [element.props.style, { height: '100%', width: '100%' }],
         })}
```

Comment:
Unrelated to this PR, but I wonder why is this needed given there's already a wrapper view. Changing the dimensions of the view may be unnecessary at best or undesirable at worst.

Reply:
I will take a look at it!

Reply:
@satya164 `React.cloneElement` has been removed 👌